src/core/elf.d

```diff
@@ -0,0 +1,255 @@
+/**
+ * This code reads ELF files and sections using memory mapped IO.
+ *
+ * Reference: http://www.dwarfstd.org/
+ *
+ * Copyright: Copyright Digital Mars 2015 - 2015.
+ * License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Yazan Dabain
+ * Source: $(DRUNTIMESRC core/elf.d)
+ */
+
+module core.elf;
```

Comment:
Perhaps this file should be located in a package so we can later add modules for Mach-O, COFF and OMF if we want to.